services/management/ia/rules_service.go

```diff
@@ -92,8 +92,22 @@ type rule struct {
 	Annotations map[string]string   `yaml:"annotations,omitempty"`
 }
 
+// RemoveVMAlertRulesFiles removes all generated rules files (*.yml) on the ia path.
+func (s *RulesService) RemoveVMAlertRulesFiles() {
```

Comment:
Let's add some tests for that function in `rules_service_test.go`. There you can find similar tests for `WriteVMAlertRulesFiles()`

Reply:
I added check that the files are delete after they are created. Please, check if it is sufficient.